src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -40,6 +40,55 @@
       "equals": true
     }
   },
+  {
+    "name": "Cache_Redis_SSL_Only",
+    "description": "Only secure connections to your Azure Cache for Redis should be enabled",
+    "recommendation": "Enable connections via SSL only to Redis Cache",
+    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#only-secure-connections-to-your-azure-cache-for-redis-should-be-enabled",
+    "evaluation": {
+      "resourceType": "Microsoft.Cache/redis",
+      "path": "properties.enableNonSslPort",
+      "equals": false
```

Comment:
Not sure what the default value is if *enableNonSslPort* is not specified.  We'll need to keep an eye on this.  This rule might have a lot of false-positives if the default value is false.

Reply:
Default (at least for API version "2020-06-01" aka latest) is false.